local-interchaintest/src/helpers/constants.rs

```diff
@@ -10,6 +11,9 @@ pub const POLYTONE_PATH: &str = "local-interchaintest/contracts/polytone";
 pub const EXECUTE_FLAGS: &str = "--gas=auto --gas-adjustment=3.0";
 pub const OSMOSIS_FEES: &str = "--fees=100000uosmo";
 
+pub const ACC0_ADDRESS_GAIA: &str = "cosmos1hj5fveer5cjtn4wd6wstzugjfdxzl0xpxvjjvr";
+pub const ACC0_ADDRESS_NEUTRON: &str = "neutron1hj5fveer5cjtn4wd6wstzugjfdxzl0xpznmsky";
+pub const ACC0_ADDRESS_OSMO: &str = "osmo1hj5fveer5cjtn4wd6wstzugjfdxzl0xpwhpz63";
```

Comment:
nit: I never added these ACC0 because we already have them in localic-utils as the CHAIN_ADMIN_ADDR, you can reuse those if you want or leave it like this

Reply:
I will reuse the ones in localic-utils! Just pushed the change.